pkgs/os-specific/linux/isolate/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, libcap, fetchFromGitHub, asciidoc }:
+
+stdenv.mkDerivation {
+  pname = "isolate";
+  version = "1.10";
+  src = fetchFromGitHub {
+    owner = "ioi";
+    repo = "isolate";
+    rev = "v1.10";
+    sha256 = "sha256-fuv9HOw0XkRBRjwAp4b6LpoB5p7a+yo66AcT3B0yQUw=";
+  };
+
+  buildInputs = [
+    stdenv.cc
+    libcap.dev
+    asciidoc
+  ];
+
+  buildPhase = ''
+    make isolate isolate.1
+  '';
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/man/man1}
+    cp ./isolate $out/bin/
+    cp ./isolate.1 $out/share/man/man1/
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

Don't tell anyone, but nested `with` is mad.